api/server.js

```diff
@@ -1,24 +1,27 @@
 const express = require("express");
-const cors = require("cors");
+const path = require("path");
+//const cors = require("cors");
```

Comment:
I commented the cors stuff out until we actually need it.  A vesitgial library setup from a tutorial I was following